plugin.cpp

```diff
@@ -114,28 +114,46 @@ static const char *default_config = QUOTE({
 		"order" : "9",
 		"displayName" : "Outstations",
 		"properties" : {
-			"linkid" : {
+				"linkid" : {
 					"description" : "The outstation link ID",
 					"displayName" : "Link ID",
 					"type" : "integer",
 					"maximum" : "65519",
 					"minimum" : "1",
 					"default" : "10"
-			},
-			"address" : {
+				},
+				"address" : {
 					"description" : "The outstation TCP address or name ",
 					"displayName" : "TCP Address",
 					"type" : "string",
 					"default" : "127.0.0.1",
 					"mandatory": "true"
 				},
-			"port" : {
+				"port" : {
 					"description" : "The outstation TCP port",
 					"displayName" : "TCP Port",
 					"type" : "integer",
 					"default" : "20000",
 					"maximum" : "65000",
 					"minimum" : "1"
+				},
+				"disableTLS": {
+					"description" : "Disable TLS",
+					"displayName" : "Disable TLS",
+					"type" : "boolean",
+					"default" : "false"
+				},
+				"TLSCAcertificate": {
```

Comment:
These two only take effect if not using the global default? It would be nice if we could set validity based on another property of the object, but I don't think we support that currently. The best we can do is clearly document this in the description of the items.